guide/ops/production-installation.md

```diff
@@ -81,6 +82,13 @@ It may be useful to use the following script to install an initial `brooklyn.pro
 % chmod 600 ~/.brooklyn/brooklyn.properties
 {% endhighlight %}
 
+### <a id="configuring-karaf-security"></a>Configuring Karaf Security
+
+Out of the box Brooklyn includes the default Karaf security configuration.
+It is used for connecting to the ssh port of Karaf (which is available
+to localhost connections only).
```

Comment:
How about:  This configuration is used manage connections to the ssh port of Karaf